bin/drush-install.sh

```diff
@@ -35,6 +35,22 @@ org.civicrm.contactsummary,\
 org.civicrm.bootstrapcivicrm,\
 org.civicrm.bootstrapcivihr
 
+##
+# Set Default localisation settings
+function set_default_localisation_settings() {
+  LOC_FILE="en_US"
+  if wget -q "https://download.civicrm.org/civicrm-l10n-core/mo/en_GB/civicrm.mo" > /dev/null; then
+    mkdir -p $1/l10n/en_GB/LC_MESSAGES/
+    mv civicrm.mo $1/l10n/en_GB/LC_MESSAGES/civicrm.mo
+    LOC_FILE="en_GB"
+  fi
+
+  UKID=$(drush cvapi Country.getsingle return="id" iso_code="GB" | grep -oh '[0-9]*')
+  drush cvapi Setting.create sequential=1 defaultCurrency="GBP" dateformatDatetime="%d/%m/%Y %l:%M %P" dateformatFull="%d/%m/%Y" dateformatFinancialBatch="%d/%m/%Y" dateInputFormat="dd/mm/yy" lcMessages=${LOC_FILE} defaultContactCountry=${UKID}
+
```

Comment:
since I will put the commands in multi-lines now , I think it is better to keep whitelines between different commands so they be easier to read.